packages/ui/src/common/components/Stepper/Stepper.tsx

```diff
@@ -34,53 +34,43 @@ export const Stepper = ({ steps, theme = StepperTheme.light }: StepperProps) =>
   const stepsToRender = asStepsToRender(steps)
 
   return (
-    <ThemeProvider theme={theme}>
-      <StepperWrap>
-        {stepsToRender.map((step, index) => (
-          <StepWrap key={index} {...step}>
```

Comment:
My understanding is that the tests were failing because the theme context is not available during the first render, so using [`waitForElement`](https://react-testing-examples.com/jest-rtl/styled-components/) on the tests which were failing would have solved the problem. But since the theme is only used within this component I figure it would be easier to just pass the theme as a prop and rewrite the styles a little.